package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@zalopay-oss/zalopay-nodejs",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
hmm, it's new api, shall we increase minor version instead of hotfix?

Reply:
@qtdan it should increase minor version. Thanks